test/external/external_test_onnx_backend.py

```diff
@@ -165,15 +177,19 @@ def supports_device(cls, device: str) -> bool:
   backend_test.exclude('test_maxpool_2d_pads_cpu')
   backend_test.exclude('test_maxpool_2d_same_lower_cpu')
 
-# Don't know how to treat special TensorProto like TensorProto.FLOAT8E4M3FN
-if getenv("CPU") or getenv("TORCH"):
-  backend_test.exclude('test_dequantizelinear_axis_cpu')
-  backend_test.exclude('test_dequantizelinear_cpu')
-
-# compiled backends cannot reshape to and from 0
+# compiled backends cannot reshape to or from 0
```

Comment:
@chenyuxyz we discussed this yesterday, we need to support 0s in the shape. torch does

```
>>> torch.zeros(10,10,10)[:, 0:0, :].shape
torch.Size([10, 0, 10])
```